### DISCUSSION TITLE
For detection of somatic hypermutation (SHM) ?

### DISCUSSION THREAD

**Discussion by xmy1990** [milaboratory/mixcr#1649] [Q&A] [answered]
Hi,
Which parameters should I use to detect somatic hypermutation (SHM) in the command "mixcr analyze exome-seq".

or can I use  "mixcr analyze exome-seq" to detect somatic hypermutation (SHM).

Thanks

**Comment by mizraelson** [comment]
Hi, could you specify what type of output you're looking for? By default, MiXCR provides the sequence of all regions that were covered for each clone. If necessary, you can customize the output to include additional columns, such as the number of mutations in a specific region or the mutations themselves.

You can find a complete list of all possible output columns in our [documentation](https://mixcr.com/mixcr/reference/mixcr-export/#common-fields).

**Comment by xmy1990** [comment]
Hi,
I want to evaluate the Somatic Hypermutation Status of IGHV status（such as 0.98） such as this paper: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7248390/

I found  the below  options in https://mixcr.com/mixcr/reference/mixcr-export/#common-fields
-vIdentityPercents     V alignment identity percents
-dIdentityPercents     D alignment identity percents
-jIdentityPercents     J alignment identity percents
-cIdentityPercents     C alignment identity percents
-vBestIdentityPercent  V best alignment identity percent
-dBestIdentityPercent  D best alignment identity percent
-jBestIdentityPercent  J best alignment identity percent
-cBestIdentityPercent  C best alignment identity percent

which of these can export the  IGHV status ?

Thanks.

**Reply by mizraelson** [reply, marked as answer]
Answered in #1678 

I suggest using:
`-nMutationsRate VRegionTrimmed`

To add this directly to `mixcr analyze` command use:

`--append-export-clones-field -nMutationsRate VRegionTrimmed`
